scripts/generate-flow-files.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+// This CLI generates .js.flow definitions from .d.ts
+// TS definitions.
+// It accepts files to generate definitions for via
+// argv;
+
+const path = require('path');
+const fs = require('fs');
+const globby = require('globby');
+
+const { promisify } = require('util');
+const { compiler, beautify } = require('flowgen');
+
+const writeFile = promisify(fs.writeFile);
+const preamble = '// @flow\n\n';
+
+const gen = async () => {
+  const cwd = process.cwd();
+
+  const input = await globby([
+    'src/*.d.ts',
+    'src/**/*.d.ts'
+  ], {
+    gitignore: true
+  });
+
+  if (input.length === 0) {
+    throw new Error('No input files passed as arguments.');
+  }
+
+  console.log(`Compiling ${input.length} TS definitions to Flow...`);
+
+  const defs = input.map(filename => {
+    const fullpath = path.resolve(cwd, filename);
+
+    let flowdef = compiler.compileDefinitionFile(fullpath);
```

Comment:
Any reason not to just wrap this in `beautify` directly? `compileDefinitionFile` appears sync, as does `beautify`. The pipe operator would be nice here 😂 

```suggestion
    const flowdef = beautify(compiler.compileDefinitionFile(fullpath));
```

Reply:
lol that was because I wasn't sure whether it was working, so I must've forgotten to clean it up after

Reply:
^ Literally me on every PR (I do a lot of force pushing later on).